.github/workflows/lint.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2024, Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+---
+name: Lint
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - "master"
+      - "[0-9]+.[0-9]+"
+      - "[0-9]+.[0-9]+-stable"
+    paths-ignore:
+      - '**/*.md'
+      - '.github/**'
+  pull_request:
+    branches:
+      - "master"
+      - "[0-9]+.[0-9]+"
+      - "[0-9]+.[0-9]+-stable"
+    paths-ignore:
+      - '**/*.md'
+      - '.github/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
If we choose to run the check on all the files in the repo, not on the files affected by the PR for which the check runs, we should be able to ensure that all the hashes are consistent when we merge this PR (#4334). Are we?

Reply:
Yep, we run go tests for the entire repo, not for the files that have been changed. It's not a problem as we try to keep the go tests green.

Reply:
Okay, I see the check is already green for this PR. Good. Document please the workaround for the current inconsistency, and that's enough (https://github.com/lf-edge/eve/pull/4334#discussion_r1797004499).